moveit_kinematics/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(moveit_kinematics)
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED cmake_modules)
```

Comment:
Uhha, you are right. It's for compiling eigen3, and it is deprecated.  I'm still using the old style.
Already remove, thanks for let me know this.